packages/webapp/src/store/api/types/index.ts

```diff
@@ -56,13 +64,24 @@ export interface AnimalBatch {
   internal_identifier: number;
   name: string | null;
   notes: string | null;
+  origin_id: number;
   photo_url: string | null;
   sex_detail: { sex_id: number; count: number }[];
+  sire: string | null;
+  organic_status: 'Non-Organic' | 'Transitional' | 'Organic';
+  supplier: string | null;
+  price: number | null;
   animal_removal_reason_id: number | null;
   removal_explanation: string | null;
   removal_date: string | null;
 }
 
+export interface PostBatchSexDetail {
+  animal_batch_sex_detail?: { sex_id: number; count: number }[];
```

Comment:
my only question is if this is to match up a name difference from frontend and backend -- is it possible to fix it to be one name?

Reply:
That's actually a bug...!!! Sex detail will not be added unless the key is `sex_detail`.
Thank you so much for pointing that out!